cmd/juju/model/config.go

```diff
@@ -473,23 +483,23 @@ func (c *configCommand) handleIsKeyOfModel(attrs config.ConfigValues, ctx *cmd.C
 				// YAML or JSON formatting, so we print out
 				// the value unadorned.
 				return nil, c.out.WriteFormatter(ctx, cmd.FormatSmart, value.Value), true
-			} else {
-				return config.ConfigValues{key: config.ConfigValue{Source: value.Source, Value: value.Value}}, nil, true
 			}
-		} else {
-			return attrs, nil, false
+			return config.ConfigValues{key: config.ConfigValue{Source: value.Source, Value: value.Value}}, nil, true
 		}
-	} else {
-		// In tabular format, don't print "cloudinit-userdata" it can be very long,
-		// instead give instructions on how to print specifically.
-		if value, ok := attrs[config.CloudInitUserDataKey]; ok && c.out.Name() == "tabular" {
-			if value.Value.(string) != "" {
-				value.Value = "<value set, see juju model-config cloudinit-userdata>"
-				attrs["cloudinit-userdata"] = value
-			}
-			return attrs, nil, true
+
+		return attrs, nil, false
+	}
+
+	// In tabular format, don't print "cloudinit-userdata" it can be very long,
+	// instead give instructions on how to print specifically.
+	if value, ok := attrs[config.CloudInitUserDataKey]; ok && c.out.Name() == "tabular" {
```

Comment:
That's really neat! Are there any other model-config bits that are also long (e.g. a cert of something like that) and could be abbreviated? If that's the case you could make a "too-long-values" list of keys and apply this logic in a loop.